doc/explanations/faq.rst

```diff
@@ -1,218 +1,337 @@
-********************************************************************************
+**Q. What are the costs related to minting and trading native tokens?**
 
-Costs related to multi assets can be divided into two categories:
+A. Costs related to multi assets can be divided into two categories:
 
-**Fees**: Sending and minting tokens affects the fees that the author of the transaction must pay. As with an ada-only ledger, the fees are calculated based on the total size of the transaction. There might also be fees for checking minting policies, but initially only multisig policies are supported, which do not incur additional fees on top of the transaction size-based ones.
+-  **Fees**: Sending and minting tokens affects the fees that the author
+   of the transaction must pay. As with an ada-only ledger, the fees are
+   calculated based on the total size of the transaction. There might
+   also be fees for checking minting policies, but initially only
+   multisig policies are supported, which do not incur additional fees
+   on top of the transaction size-based ones.
 
-**Min-Ada-Value**: Every output created by a transaction must include a minimum amount of ada, which is calculated based on the size of the output (that is, the number of different token types in it, and the lengths of their names).
+-  **Min-ada-value**: Every output created by a transaction must include
+   a minimum amount of ada, which is calculated based on the size of the
+   output (that is, the number of different token types in it, and the
+   lengths of their names).
 
-Recall that outputs may contain a heterogeneous collection of tokens, including ad Ada is a limited resource in the Cardano system. Requiring some amount of ada be included in every output on the ledger (where that amount is based on the size of the output, in bytes) protects the size of the Cardano ledger from growing intractably.
-For a detailed explanation of the min-ada-value, see "Minimum Ada Value Requirement".
+**Min-ada-value explanation:**
 
+Remember that outputs may contain a heterogeneous collection of tokens,
+including ada is a limited resource in the Cardano system. Requiring
+some amount of ada be included in every output on the ledger (where that
+amount is based on the size of the output, in bytes) protects the size
+of the Cardano ledger from growing intractably.
 
-What types of assets can I use to cover costs associated with native tokens?
-************************************************************************************
+**Q. What types of assets can I use to cover costs associated with
+native tokens?**
 
-Currently, only ada can be used to make fee payments or to satisfy the min-ada-value constraint.
+A. Currently, only ada can be used to make fee payments or deposits.
 
-How does coin selection work for custom native tokens?
-********************************************************************************
+**Q. How does coin selection work for custom native tokens?**
```

Comment:
coin selection really has nothing to do with the ledger, but really a wallet implementation detail.

Reply:
deleted